### PR TITLE
Update RSA8e for 0.9

### PR DESCRIPTION
**Pull request by ricardopereira**

### CODE REVIEW

File: Source/ARTAuth.h [hunk 1]
```diff
@@ -43,13 +43,17 @@ ART_ASSUME_NONNULL_BEGIN
  */
 - (void)requestToken:(art_nullable ARTTokenParams *)tokenParams withOptions:(art_nullable ARTAuthOptions *)authOptions
             callback:(void (^)(ARTTokenDetails *__art_nullable, NSError *__art_nullable))callback;
+- (void)requestToken:(void (^)(ARTTokenDetails *__art_nullable, NSError *__art_nullable))callback;
```
Comment (ricardopereira) [inline Source/ARTAuth.h]: Objective-C doesn't allow optional values on arguments, so we need to have another method.

Comment (mattheworiordan) [inline Source/ARTAuth.h]: Ok, so this is overloaded?

Comment (ricardopereira) [inline Source/ARTAuth.h]: Yes.

Comment (ricardopereira) [inline Source/ARTAuth.h]: Yes.

**Pull request merged**